### PR TITLE
:seedling: Detect more information about runtime

### PR DESCRIPTION
**Pull request by mudler**
This introduces a `system` and a `kairos` block available in kairos-agent get state.

This allows for instance to query the agent for the kairos version as such:

```bash
$ kairos-agent state get kairos.version
v1.5.1
$ kairos-agent state get kairos.flavor
opensuse-tumbleweed
$ kairos-agent state get system.os.vendor
kairos
```

Part of #755

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -25,10 +25,12 @@ require (
 	github.com/pterm/pterm v0.12.54
 	github.com/qeesung/image2ascii v1.0.1
 	github.com/santhosh-tekuri/jsonschema/v5 v5.2.0
+	github.com/spectrocloud-labs/herd v0.3.0
```
Comment (Itxaka) [inline go.mod:28]: Not sure why herd is pulled, I can't see it used anywhere?

Comment (mudler) [reply go.mod:28]: no idea to be honest.. good point, I've just run go mod tidy..

Comment (mudler) [reply go.mod:28]: running it again doesn't remove it, so must be throwed in by something

Comment (mudler) [reply go.mod:28]: ah, maybe we bumped yip, that uses herd now

Comment (Itxaka) [reply go.mod:28]: Ha, interesting, I would expect it to be marked as indirect. The more I know about go deps the less I understand them lol

**Review by Itxaka — APPROVED** [review]

**Review by Itxaka — APPROVED** [review]

**Pull request merged**